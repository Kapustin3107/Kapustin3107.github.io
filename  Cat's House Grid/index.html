<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	<title>Cats House</title>
	<link href="https://fonts.googleapis.com/css?family=Roboto+Slab:400,700&display=swap" rel="stylesheet">
	<link rel="stylesheet" href="CSS/normalize.css">
	<link rel="stylesheet" href="LESS/main.css">
</head>
<body>
	<header class="main_header">
		<div class="container grid">
			<div class="header_logo">
				<img src="IMG/header-logo.png" alt="">
			</div>
			<div class="header_descr">
				<p>Изготовление домов <br> для домашних питомцев</p>
			</div>
			<div class="header_social">
				<p>Мы в социальных сетях:</p>
				<ul class="header_social__links">
					<li><a href="#"><img src="IMG/vk.png" alt="vk.com"></a></li>
					<li><a href="#"><img src="IMG/fb.png" alt="facebook.com"></a></li>
					<li><a href="#"><img src="IMG/insta.png" alt="instagramm"></a></li>
					<li><a href="#"><img src="IMG/ok.png" alt="ok.com"></a></li>
				</ul>
			</div>
			<div class="header_contacts">
				<ul class="header_contacts__list">
					<li>Есть вопросы? Звоните!</li>
					<li>+7 916 284 20 20</li>
					<li>+7 916 841 83 83</li>
					<li><a href="#">Заказать звонок</a></li>
					<li>с 9:00 до 19:00 без выходных</li>
				</ul>
			</div>
		</div>
	</header>
	<!-- <section class="main_form">
		<div class="container grid">
			
			<div class="main_form__title">
				<h2>Порадуй питомца <br> личной недвижимостью!</h2>
				<p>Доставка до квартиры <br> по всей России бесплатно!</p>
			</div>
			<div class="main_form__field">
				<h3>Получите бесплатную консультацию для подбора домика для питомца!</h3>
				<p>Подберите домик с учетом:</p>
				<ul>
					<li>- размеров изделия;</li>
					<li>- цвета;</li>
					<li>- материалов исполнения.</li>
				</ul>
				<form>
					<input type="text" name="name" id="name" placeholder="Ваше имя">
					<input type="text" name="phone" id="phone" placeholder="Ваш телефон">
					<button class="btn" type="submit">Оставить заявку</button>
				</form>
			</div>
		</div>
	</section> -->
	<!-- <section class="feature_block">
		<div class="container feature_items grid">
			<h2>Домики для питомцев украсят ваш интерьер</h2>
			<div class="feature_item">
				<img src="IMG/Feature-1.png" alt="Feature-1">
				<p>Экологически чистые <br> материалы</p>
			</div>
			<div class="feature_item">
				<img src="IMG/Feature-2.png" alt="Feature-2">
				<p>Эксклюзивный <br> дизайн</p>
			</div>
			<div class="feature_item">
				<img src="IMG/Feature-3.png" alt="Feature-3">
				<p>Бесплатный <br> макет</p>
			</div>
			<div class="feature_item">
				<img src="IMG/Feature-4.png" alt="Feature-4">
				<p>Доставка <br> по России</p>
			</div>
			<div class="feature_item">
				<img src="IMG/Feature-5.png" alt="Feature-5">
				<p>Всегда <br> в наличии</p>
			</div>
			<div class="feature_item">
				<img src="IMG/Feature-6.png" alt="Feature-6">
				<p>Отгрузка <br> в день заказа</p>
			</div>
		</div>
	</section> -->
	<!-- <section class="popular_models">
		<div class="container grid">
			<div class="popular_models__title">
				<h2>Всегда в наличии самые популярные модели</h2>
			</div>
			<button class="slider_btn slider_prew"></button>
			<div class="popular_models__item">
				<img src="IMG/pop-item-2.jpg" alt="cat">
				<button class="btn">Заказать</button>
				<span class="popular_models__price">10 000 руб.</span>
			</div>
			<div class="popular_models__item">
				<img src="IMG/pop-item-2.jpg" alt="cat">
				<button class="btn">Заказать</button>
				<span class="popular_models__price">20 000 руб.</span>
			</div>
			<div class="popular_models__item">
				<img src="IMG/pop-item-2.jpg" alt="cat">
				<button class="btn">Заказать</button>
				<span class="popular_models__price">10 000 руб.</span>
			</div>
			<button class="slider_btn slider_next"></button>
			<div class="popular_models__form">
				<h3>Вы можете изменить цвет и размер любой из моделей!</h3>
				<p>Заполните форму и узнайте подробности!</p>
				<form>
					<input type="text" name="name" id="name" placeholder="Ваше имя">
					<input type="text" name="phone" id="phone" placeholder="Ваш телефон">
					<button class="btn" type="submit">Узнать подробности</button>
				</form>
			</div>
		</div>
	</section> -->
	<!-- <section class="design_items">
		<div class="container grid">
			<div class="design_items__title">
				<h2>Более 35% клиентов приходят к нам по рекомендациям!</h2>
			</div>
			<div class="design_items__unit">
				<img src="IMG/design-unit-1.jpg" alt="design-1">
				<p>Вы можете заказать домик <br> по собственным эскизам <br> или фотографиям</p>
				<a href="#" class="btn">Заказать по эскизам</a>
			</div>
			<div class="design_items__unit">
				<img src="IMG/design-unit-2.jpg" alt="design-2">
				<p>Выберите подушечку</p>
				<a href="#" class="btn">Заказать по эскизам</a>
			</div>
			<div class="design_items__unit">
				<img src="IMG/design-unit-3.jpg" alt="design-3">
				<p>Сделайте гравировку <br>имени питомца на домике </p>
				<a href="#" class="btn">Заказать по эскизам</a>
			</div>
		</div>
	</section> -->
	<!-- <section class="feedback">
		<div class="container grid">
			<div class="feedback_title">
				<h2>Более 35% клиентов приходят к нам по рекомендациям!</h2>
			</div>
			<div class="feedback_icon icon_1">
				<img src="IMG/feedback-1.jpg" alt="cat-1">
			</div>
			<div class="feedback_text feedback_1">
					<h3>Имя Фамилия</h3>
					<span>г. Город</span>
					<p>Lorem Ipsum - это текст-"рыба", часто используемый в печати и вэб-дизайне. Lorem Ipsum является стандартной "рыбой" для текстов на латинице с начала XVI века. В то время некий безымянный печатник создал большую коллекцию размеров и форм шрифтов, используя Lorem Ipsum для распечатки образцов. </p>
			</div>
			<div class="feedback_text feedback_2">
					<h3>Имя Фамилия</h3>
					<span>г. Город</span>
					<p>Lorem Ipsum - это текст-"рыба", часто используемый в печати и вэб-дизайне. Lorem Ipsum является стандартной "рыбой" для текстов на латинице с начала XVI века. В то время некий безымянный печатник создал большую коллекцию размеров и форм шрифтов, используя Lorem Ipsum для распечатки образцов. </p>
			</div>
			<div class="feedback_icon icon_2">
					<img src="IMG/feedback-2.jpg" alt="cat-2">
			</div>
			<div class="feedback_more">
				<a href="#" class="btn">Еще отзывы</a>
			</div>
		</div>
	</section> -->
	<!-- <section class="order_form">
		<div class="container grid">
			<div class="order_form__title">
				<h2>Как заказать</h2>
			</div>
			<div class="order_form__feature">
				<img src="../ Cat's House Grid/IMG/order-feature-1.png" alt="">
				<p>Наш менеджер <br>связывается <br> с вами по телефону </p>
			</div>
			<div class="order_form__feature">
				<img src="../ Cat's House Grid/IMG/order-feature-2.png" alt="">
				<p>Вы оставляете заявку</p>
			</div>
			<div class="order_form__feature">
				<img src="../ Cat's House Grid/IMG/order-feature-3.png" alt="">
				<p>Мы уточняем <br> параметры <br> Вашего заказа </p>
			</div>
			<div class="order_form__feature">
				<img src="../ Cat's House Grid/IMG/order-feature-4.png" alt="">
				<p>Мы отправляем <br> Вам посылку!</p>
			</div>
			<div class="order_form_field">
				<form>
					<input type="text" name="name" id="name" placeholder="Ваше имя">
					<input type="text" name="phone" id="phone" placeholder="Ваш телефон">
					<button class="btn" type="submit">Узнать подробности</button>
				</form>
				<p>С вами свяжется менеджер в течении 14 минут.</p>
				<h3>Или перезвоните по телефону: <br>8 (495) 555-55-55</h3>
			</div>
		</div>
	</section> -->
	<!-- <footer class="main_footer">
		<div class="container grid">
			<div class="header_logo">
				<img src="IMG/header-logo.png" alt="">
			</div>
			<div class="header_descr">
				<p>Изготовление домов для домашних питомцев</p>
			</div>
			<div class="header_social">
				<p>Мы в социальных сетях:</p>
				<ul class="header_social__links">
					<li><a href="#"><img src="IMG/vk.png" alt="vk.com"></a></li>
					<li><a href="#"><img src="IMG/fb.png" alt="facebook.com"></a></li>
					<li><a href="#"><img src="IMG/insta.png" alt="instagramm"></a></li>
					<li><a href="#"><img src="IMG/ok.png" alt="ok.com"></a></li>
				</ul>
			</div>
			<div class="header_contacts">
				<ul class="header_contacts__list">
					<li>Есть вопросы? Звоните!</li>
					<li>+7 916 284 20 20</li>
					<li>+7 916 841 83 83</li>
					<li><a href="#">Заказать звонок</a></li>
					<li>с 9:00 до 19:00 без выходных</li>
				</ul>
			</div>
		</div>
	</footer> -->

	

	
</body>
</html>