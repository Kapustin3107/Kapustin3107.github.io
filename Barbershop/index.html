<!DOCTYPE html>
<html lang="ru">
<head>
	<meta charset="UTF-8">
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	<title>Barbershop Borodinski</title>
	<link href="https://fonts.googleapis.com/css?family=PT+Sans+Narrow:400,700&display=swap&subset=cyrillic" rel="stylesheet">
	<link rel="stylesheet" href="css/main.css">
</head>
<body>
	<header class="main-header">
		<div class="container main-header_wrapper">
			<div class="header_mobile-logo"></div>
			<div class="header_mobile-open">
				<div class="menu_mobile-hidden">
				</div>
				<span>ГЛАВНАЯ</span>
				<div class="menu_mobile-closed">
					<button class="menu_opener"></button>
				</div>
			</div>
			<ul class="main-header_menu">
				<li><a href="#">Главная</a></li>
				<li><a href="#">Наши работы</a></li>
				<li><a href="#">Записаться</a></li>
				<li><a href="#">Контакты</a></li>
				<li><a href="#">HTML Academy</a></li>
				<li ><a class="menu-login" href="#">
					<svg height="10pt" viewBox="0 0 512 512" width="10pt" xmlns="http://www.w3.org/2000/svg" fill="#fff"><path d="m453.332031 512h-224c-32.363281 0-58.664062-26.304688-58.664062-58.667969v-96c0-8.832031 7.167969-16 16-16s16 7.167969 16 16v96c0 14.699219 11.964843 26.667969 26.664062 26.667969h224c14.699219 0 26.667969-11.96875 26.667969-26.667969v-394.664062c0-14.699219-11.96875-26.667969-26.667969-26.667969h-224c-14.699219 0-26.664062 11.96875-26.664062 26.667969v96c0 8.832031-7.167969 16-16 16s-16-7.167969-16-16v-96c0-32.363281 26.300781-58.667969 58.664062-58.667969h224c32.363281 0 58.667969 26.304688 58.667969 58.667969v394.664062c0 32.363281-26.304688 58.667969-58.667969 58.667969zm0 0"/><path d="m325.332031 272h-309.332031c-8.832031 0-16-7.167969-16-16s7.167969-16 16-16h309.332031c8.832031 0 16 7.167969 16 16s-7.167969 16-16 16zm0 0"/><path d="m240 357.332031c-4.097656 0-8.191406-1.554687-11.308594-4.691406-6.25-6.25-6.25-16.382813 0-22.636719l74.027344-74.023437-74.027344-74.027344c-6.25-6.25-6.25-16.386719 0-22.636719 6.253906-6.25 16.386719-6.25 22.636719 0l85.332031 85.335938c6.25 6.25 6.25 16.382812 0 22.632812l-85.332031 85.332032c-3.136719 3.160156-7.230469 4.714843-11.328125 4.714843zm0 0"/></svg><span class="login_btn">Войти</span></a></li>
			</ul>

		</div>
	</header>
	<main>
		<section class="logo_section">
			<h1>Barbershop Borodinski</h1>
		</section>
		<section class="advantages">
			<div class="container">
				<div class="advantages_wrapper">
					<div class="advantages_header">
						<h2>Стрижка у нас <br> это выгодно!</h2>
						<p>Мужская стрижка требует точного подхода. Обратимся к статистике:</p>
						<p class="advantages_header-explanation">* — приведённые данные ложь</p>
					</div>
					<div class="advantages_table-item item-1">
						<strong>1 500<sup>*</sup> </strong>
						<p>рублей вложений</p>
					</div>
					<div class="advantages_table-item item-2">
						<strong>7 200<sup>*</sup> </strong>
						<p>секунд времени мастера</p>
					</div>
					<div class="advantages_table-item item-3">
						<strong>90 000<sup>*</sup> </strong>
						<p>рублей вложений</p>
					</div>
					<div class="advantages_table-item item-4">
						<strong>500 000<sup>*</sup> </strong>
						<p>рублей вложений</p>
					</div>
				</div>
				<div class="advantages_second-wrapper">
					<div class="advantages_second-item second_item-1">
						<div class="item_bg bg-1"></div>
						<div class="item-title">
							<h3>БЫСТРО</h3>
							<p>Мы делаем свою работу быстро! Два часа пролетят незаметно и вы — счастливый обладатель стильной стрижки-минутки</p>
						</div>
					</div>
					<div class="advantages_second-item second_item-2">
						<div class="item_bg bg-2"></div>
						<div class="item-title">
							<h3>КРУТО</h3>
							<p>Забудьте, как вы стриглись раньше. Мы сделаем из вас звезду футбола или кино. Во всяком случае внешне.</p>
						</div>
						
					</div>
					<div class="advantages_second-item second_item-3">
						<div class="item_bg bg-3"></div>
						<div class="item-title">
							<h3>ДОРОГО</h3>
							<p>Наши мастера — профессионалы своего делаи не могут стоить дёшево. К тому же, разве цена не даёт определённый статус?</p>
						</div>
					</div>
				</div>
			</div>
		</section>
	</main>
</body>
</html>