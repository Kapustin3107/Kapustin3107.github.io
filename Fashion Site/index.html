<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<title>Fashion</title>
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	<link href="https://fonts.googleapis.com/css?family=Montserrat:300,400,600,700,800" rel="stylesheet">
	<link rel="stylesheet" href="css/normalize.css.css">
	<link rel="stylesheet" href="css/main.css">
</head>
<body>
	<header class="main_header">
		<div class="main_header__wrapper">
			<div class="main_heade__logo">
				<h1>Fa–µshion</h1>
				<p>Tagline here</p>
			</div>
			<div class="main_header__nav">
				<ul>
					<li><a href="#">Home</a></li>
					<li><a href="#">Shop</a></li>
					<li><a href="#">Woman</a></li>
					<li><a href="#">Man</a></li>
					<li><a href="#">Find stores</a></li>
				</ul>
			</div>
			<div class="main_header__user">
				<ul>
					<li><a href="#"><img src="img/cart.png" alt="cart"></a></li>
					<li><a href="#"><img src="img/search .png" alt="search"></a></li>
					<li><a href="#"><img src="img/icon.png" alt="menu button"></a></li>
				</ul>
			</div>
		</div>
	</header>
	
</body>
</html>