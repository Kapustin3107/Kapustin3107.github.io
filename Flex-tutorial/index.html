<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	<title>Flex-grid</title>
	<link href="https://fonts.googleapis.com/css?family=Roboto:300,400,700&display=swap" rel="stylesheet">
	<link rel="stylesheet" href="css/main.css">
</head>
<body>


	<header class="header">
		<div class="header__image ibg">
			<img src="img/Rectangle 1.jpg" alt="First pictire">
			<div class="header__logo">
				
			</div>
		</div>
		<div class="header__content">
			<div class="header__btn">
				<span></span>
			</div>
			<div class="header__info">
				<h1 class="info__title">
					<span>Ali Sayed’s</span>
					<span>Web Designing</span>
					<span>Project</span>
				</h1>
				<p class="info__text">Lorem ipsum dolor sit amet, consectetuer adipiscing elit, sed diam nonummy nibh euismod tincidunt ut laoreet dolore</p>
				<p class="info__text">quis nostrud exerci tation ullamcorper suscipit lobortis nisl ut aliquip ex ea commodo consequat. Duis autem vel eum iriure dol.</p>
			</div>
		</div>
	</header>

	<main class="description">
		<div class="description__left ibg">
			<img src="img/Rectangle 2.jpg" alt="Secong picture">
			<h2 class="info__title"><span>Drawing</span> Scketch</h2>
			<p class="info__text">Lorem ipsum dolor sit amet, consectetuer adipiscing elit, sed diam nonummy nibh euismod quis nostrud exerci tation ullamcorper suscipit lobortis nisl ut aliquip ex consectetuer adipiscing elit, sed diam nonummy nibh euismod tincidu.</p>
			<h2 class="info__title"><span>Making</span> Design</h2>
			<p class="info__text">Lorem ipsum dolor sit amet, consectetuer adipiscing elit, sed diam nonummy nibh euismod quis nostrud exerci tation ullamcorper suscipit lobortis nisl ut aliquip ex consectetuer adipiscing elit, sed diam nonummy nibh euismod tincidu.</p>
			<h2 class="info__title"><span>Developing</span> Product</h2>
			<p class="info__text">Lorem ipsum dolor sit amet, consectetuer adipiscing elit, sed diam nonummy nibh euismod quis nostrud exerci tation ullamcorper suscipit lobortis nisl ut aliquip ex consectetuer adipiscing elit, sed diam nonummy nibh euismod tincidu.</p>
		</div>
		<div class="description__right">
			<div class="right__top ibg">
				<img src="img/Rectangle 3.jpg" alt="Third picture">
				<h2 class="info__title"><span>Best</span> Strarting</h2>
				<p class="info__text">Lorem ipsum dolor sit amet, quis nostrud exerci tation ullamcorper suscipit lobortis nisl ut aliquip ex  consectetuer adipiscing elit, sed diam nonummy nibh euismod tincidunt ut laoreet dolore.quis nostrud exerci tation ullamcorper suscipit lobortis nisl ut aliquip ex ea.</p>
				<a href="#" class="page__btn">View Our History</a>
			</div>
			<div class="right__bottom ibg">
				<img src="img/Rectangle 4.jpg" alt="Fourth picture">
				<h2 class="info__title"><span>Verified</span> Dummy</h2>
				<p class="info__text">Lorem ipsum dolor sit amet, quis nostrud exerci tation ullamcorper suscipit lobortis nisl ut aliquip ex  consectetuer adipiscing elit, sed diam nonummy nibh euismod tincidunt ut laoreet dolore.quis nostrud exerci tation ullamcorper suscipit lobortis nisl ut aliquip ex ea.</p>
				<a href="#" class="page__btn">Go To The Page</a>
			</div>
		</div>
	</main>

	<footer class="footer">
		<div class="footer__copyright">
			<h2 class="info__title"><span>This Webpage Is </span> <br>Designed By</h2>

			<div class="copyright__logo">
			</div>
			
			<h3 class="info__title">Ali Sayed</h3>
			<p>Graphic Designer</p>
		</div>
		<div class="footer__form ibg">
			<img src="img/Rectangle 6.jpg" alt="Six picture">
			<h2 class="info__title"><span>Say</span> ‘Hello’</h2>
			<form action="#">
				<fieldset>
					<input type="text" name="name" id="name" placeholder="Your name">
					<input type="text" name="email" id="email" placeholder="Your email">
					<textarea name="text" id="text" cols="30" rows="10" placeholder="Your Message Here"></textarea>
				</fieldset>
			</form>
			<div class="socials">
				<p>Stay Connnected With Us...</p>
				<div class="socials__links">
					<a class="facebook" href="#">facebook</a>
					<a class="dribble" href="#">insta</a>
					<a class="twitter" href="#">twitter</a>
				</div>
				<p>Please Feel Free To Contat With me For Any Kind of Prject.</p>
			</div>
		</div>
	</footer>








	<script src="https://code.jquery.com/jquery-3.4.1.min.js"
  integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo="
  crossorigin="anonymous">
  	
  </script>

	<script>
		function ibg(){

$.each($('.ibg'), function(index, val) {
if($(this).find('img').length>0){
$(this).css('background-image','url("'+$(this).find('img').attr('src')+'")');
}
});
}

ibg();
	</script>
</body>
</html>