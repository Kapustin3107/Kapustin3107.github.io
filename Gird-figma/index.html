<!DOCTYPE html>
<html lang="en" class="page">
  <head>
    <meta charset="UTF-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>Figma grid template</title>
    <link rel="stylesheet" href="css/style.css" />
  </head>
  <body class="page__body">
    <div class="page__wrapper">
      <header class="header">
        <div class="container">
          <div class="header__wrapper">
            <div class="header__logo">
              <button class="header__burger">
                <span></span>
              </button>
              <span>Grid</span>
            </div>
            <nav class="header__menu">
              <ul class="menu__list">
                <li class="menu__item">
                  <a class="menu__link current" href="#">How it works</a>
                </li>
                <li class="menu__item">
                  <a class="menu__link" href="#">Who we are</a>
                </li>
                <li class="menu__item">
                  <a class="menu__link" href="#">What we do</a>
                </li>
                <li class="menu__item">
                  <a class="menu__link" href="#">Contact us</a>
                </li>
              </ul>
            </nav>
            <button type="button" class="header__btn btn">Sign In</button>
          </div>
        </div>
      </header>

      <main>
        <section class="images">
          <div class="container">
            <div class="images__title">
              <h2 class="images__big">
                <b>Combine</b>
                <span> cool </span>
                images
              </h2>
              <p>To represent a product</p>
            </div>
            <div class="images__grig grid">
              <div class="grid__sidebar">
                <p>
                  Use mixed grid with imagery, replace with your own photos and
                  descriptions
                </p>
                <img src="images/png/Card-1.png" alt="Images card 1" />
                <a href="#" class="grid__link btn">Learn more</a>
              </div>

              <div class="grid__item grid__imageblock">
                <a href="#"
                  ><img
                    width="470px"
                    src="images/png/Card-2.png"
                    alt="Images card 2"
                /></a>

                <div class="grid__textgroup">
                  <p textgroup__info>
                    This is muplipurpose grid, it fits for portfolio, sevices or
                    agency wen site
                  </p>
                  <div class="textgroup__img">
                    <a href="#"
                      ><img
                        width="170px"
                        src="images/png/Card-3.png"
                        alt="Images card 3"
                    /></a>
                    <a href="#"
                      ><img src="images/png/Card-4.png" alt="Images card 4"
                    /></a>
                  </div>
                </div>
              </div>

              <div class="grid__item grid__imageblock">
                <a href="#"
                  ><img src="images/png/Card-5.png" alt="Images card 5"
                /></a>
                <a href="#"
                  ><img src="images/png/Card-6.png" alt="Images card 6"
                /></a>
                <a href="#"
                  ><img src="images/png/Card-7.png" alt="Images card 7"
                /></a>
              </div>
            </div>
          </div>
        </section>

        <section class="features">
          <div class="container">
            <div class="features__title">
              <h3>Delivering good designs since 1954 ðŸššðŸ’¨</h3>
              <p>
                We're the first multi-purpose design kit solutions for
                businesses. We help you bridge <br />
                gaps between your layouts, and developers to empower all
                involved.
              </p>
            </div>
            <div class="features__wrapper">
              <div class="feature">
                <span class="feature__icon"></span>
                <h4>Support</h4>
                <p>
                  Delivering faster and more personalized support with shared
                  screens and cool design systems for Figma
                </p>
                <button class="feature__btn btn">Learn more</button>
              </div>
              <div class="feature">
                <span class="feature__icon"></span>
                <h4>Sales growth</h4>
                <p>
                  Identify qualified customers with easy-to-use live chat
                  messaging and AI-based Sales Bot
                </p>
                <button class="feature__btn btn">Learn more</button>
              </div>
              <div class="feature">
                <span class="feature__icon"></span>
                <h4>Coponents-driven</h4>
                <p>
                  Delivering faster and more personalized support with shared
                  screens and cool design systems for Figma
                </p>
                <button class="feature__btn btn">Learn more</button>
              </div>
              <div class="feature">
                <span class="feature__icon"></span>
                <h4>Swap the icon</h4>
                <p>
                  You can toggle to any icon within Instances and customize
                  outlined stroke to more bolder or lighter
                </p>
                <button class="feature__btn btn">Learn more</button>
              </div>
            </div>
          </div>
        </section>

        <section class="pricing">
          <div class="container">
            <div class="pricing__title">
              <h2>Affordable pricing</h2>
              <h4>
                Bill me
                <button class="active">monthly</button>
                <span>&#8226;</span>
                <button>yearly</button>
              </h4>
            </div>
            <div class="pricing__wrapper">
              <div class="price">
                <div class="price__type">
                  <span>Like a pussy</span>
                </div>
                <div class="price__amount">
                  <p><span>Free</span> / forever</p>
                </div>
                <ul class="price__features">
                  <li><span></span>Components-driven system</li>
                  <li><span></span>Sales-boosting landing pages</li>
                  <li><span></span>Awesome Feather icons pack</li>
                </ul>
                <button type="button" class="price__btn btn">
                  Try it for free
                </button>
              </div>
              <div class="price">
                <div class="price__type">
                  <span>Individual</span>
                </div>
                <div class="price__amount">
                  <p><span>&#8364;24</span> / month</p>
                </div>
                <ul class="price__features">
                  <li><span class="yellow"></span>Components-driven system</li>
                  <li>
                    <span class="yellow"></span>Sales-boosting landing pages
                  </li>
                  <li>
                    <span class="yellow"></span>Awesome Feather icons pack
                  </li>
                  <li>
                    <span class="yellow"></span>Themed into 3 different styles
                  </li>
                  <li><span class="yellow"></span>Will help to learn Figma</li>
                </ul>
                <button type="button" class="price__btn btn">
                  Regular licence
                </button>
              </div>
              <div class="price">
                <div class="price__type">
                  <span>Professional</span>
                </div>
                <div class="price__amount">
                  <p><span>&#8364;12</span> / editor</p>
                </div>
                <ul class="price__features">
                  <li><span class="green"></span>Components-driven system</li>
                  <li>
                    <span class="green"></span>Sales-boosting landing pages
                  </li>
                  <li><span class="green"></span>Awesome Feather icons pack</li>
                  <li>
                    <span class="green"></span>Themed into 3 different styles
                  </li>
                </ul>
                <button type="button" class="price__btn btn">
                  Extended licence
                </button>
              </div>
            </div>
          </div>
        </section>
      </main>

      <footer class="footer">
        <div class="container">
          <div class="footer__wrapper">
            <div class="footer__socials">
              <span class="footer__title">Follow us</span>
              <ul class="socials__list">
                <li><a class="facebook" href="#"></a>Facebook</li>
                <li><a class="linkedin" href="#"></a>LinkedIn</li>
                <li><a class="slack" href="#"></a>Slack</li>
                <li><a class="twitter" href="#"></a>Twitter</li>
              </ul>
            </div>
            <div class="footer__email">
              <span class="footer__title">Follow us</span>
              <a href="mailto:hellow@fapsterapp.com">hellow@fapsterapp.com</a>
            </div>
            <div class="footer__info">
              <span class="footer__title">Information</span>
              <ul class="info__list">
                <li><a href="#">About Fapster app</a></li>
                <li><a href="#">Onhovered / Active</a></li>
                <li><a href="#">We are hiring</a></li>
                <li><a href="#">Privacy policy</a></li>
                <li><a href="#">Get in touch</a></li>
                <li><a href="#">Term of service</a></li>
                <li><a href="#">Resources</a></li>
                <li><a href="#">Things we like</a></li>
              </ul>
            </div>
            <div class="footer__form">
              <span class="footer__title">Keep in Touch</span>
              <form action="#">
                <input
                  type="text"
                  name="name"
                  id="name"
                  placeholder="Your name"
                />
                <input
                  type="text"
                  name="email"
                  id="email"
                  placeholder="Email"
                />
                <textarea
                  name="message"
                  id="message"
                  cols="30"
                  rows="3"
                  name="message"
                ></textarea>
                <button type="submit" class="form__btn btn">Send</button>
              </form>
            </div>
          </div>
        </div>
      </footer>
    </div>
    <script src="js/main.js"></script>
  </body>
</html>
