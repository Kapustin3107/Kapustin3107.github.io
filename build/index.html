<!doctype html><html lang="en"><head><meta charset="utf-8"/><link rel="icon" href="/Kapustin3107.github.io/Sparks/favicon.ico"/><meta name="viewport" content="width=device-width,initial-scale=1"/><meta name="theme-color" content="#000000"/><meta name="description" content="Web site created using create-react-app"/><link rel="apple-touch-icon" href="/Kapustin3107.github.io/Sparks/logo192.png"/><link rel="manifest" href="/Kapustin3107.github.io/Sparks/manifest.json"/><title>Spark stars</title><script defer="defer" src="/Kapustin3107.github.io/Sparks/static/js/main.13c5046c.js"></script><link href="/Kapustin3107.github.io/Sparks/static/css/main.41e80bfe.css" rel="stylesheet"></head><body><noscript>You need to enable JavaScript to run this app.</noscript><div id="root"></div></body></html>