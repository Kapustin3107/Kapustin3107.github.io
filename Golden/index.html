<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>Golden</title>
    <link rel="stylesheet" type="text/css" href="slick/slick.css"/>
    <link rel="stylesheet" type="text/css" href="slick/slick-theme.css"/>
    <link rel="stylesheet" href="css/normalize.css">
    <link rel="stylesheet" href="css/main.css">
</head>
<body>
    <header class="main_header">
        <div class="container flex main_header__wrapper">
            <div class="main_header__logo">
                    <img src="./img/logo.png" alt="Golden logo">
                    <div class="main_header__mobile">
                            <button>
                                <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 50 50" version="1.1" width="50px" height="50px" fill="white">
                                    <g id="surface1">
                                    <path style=" " d="M 0 7.5 L 0 12.5 L 50 12.5 L 50 7.5 Z M 0 22.5 L 0 27.5 L 50 27.5 L 50 22.5 Z M 0 37.5 L 0 42.5 L 50 42.5 L 50 37.5 Z "/>
                                    </g>
                                </svg>
                                        
                            </button>
                        </div>
            </div>
            <div class="main_header__menu">
                <ul class="flex">
                    <li><a href="#">Home</a></li>
                    <li><a href="#">Services</a></li>
                    <li><a href="#">Porfolio</a></li>
                    <li><a href="#">About</a></li>
                    <li><a href="#">Contact</a></li>
                </ul>
            </div>
            
        </div>
        <div class="main_header__titles container">
            <h1>Welcome To Our Studio!</h1>
            <span>Itâ€™s nice to Meet you</span>
            <a class="btn" href="#">Tell Me More</a>
        </div>
    </header>

    <section class="services">
        <div class="container">
                <div class="services_title">
                    <h1>Services</h1>
                    <p>Proin iaculis purus consequat sem cure.</p>
                </div>
                <div class="services_items flex">
                        <div class="services_item">
                            <img src="./img/icon1.png" alt="">
                            <h3>E-Commerce</h3>
                            <p>Proin iaculis purus consequat sem cure 
                                digni ssim. Donec porttitora entum suscipit 
                                aenean rhoncus posuere odio in tincidunt.</p>
                        </div>
                            
                        <div class="services_item">
                            <img src="./img/icon2.png" alt="">
                            <h3>Responsive Web</h3>
                            <p>Proin iaculis purus consequat sem cure 
                                digni ssim. Donec porttitora entum suscipit 
                                aenean rhoncus posuere odio in tincidunt.</p>
                        </div>

                        <div class="services_item">
                            <img src="./img/icon3.png" alt="">
                            <h3>Web Security</h3>
                            <p>Proin iaculis purus consequat sem cure 
                                digni ssim. Donec porttitora entum suscipit 
                                aenean rhoncus posuere odio in tincidunt.</p>
                        </div>

                </div>
                <div class="services_items__mobile flex">
                        <div class="services_item">
                            <img src="./img/icon1.png" alt="">
                            <h3>E-Commerce</h3>
                            <p>Proin iaculis purus consequat sem cure 
                                digni ssim. Donec porttitora entum suscipit 
                                aenean rhoncus posuere odio in tincidunt.</p>
                        </div>
                            
                        <div class="services_item">
                            <img src="./img/icon2.png" alt="">
                            <h3>Responsive Web</h3>
                            <p>Proin iaculis purus consequat sem cure 
                                digni ssim. Donec porttitora entum suscipit 
                                aenean rhoncus posuere odio in tincidunt.</p>
                        </div>

                        <div class="services_item">
                            <img src="./img/icon3.png" alt="">
                            <h3>Web Security</h3>
                            <p>Proin iaculis purus consequat sem cure 
                                digni ssim. Donec porttitora entum suscipit 
                                aenean rhoncus posuere odio in tincidunt.</p>
                        </div>

                </div>

        </div>
    </section>

    <section>

    </section>

    <section>

    </section>

    <footer>

    </footer>











<script type="text/javascript" src="//code.jquery.com/jquery-1.11.0.min.js"></script>
<script type="text/javascript" src="//code.jquery.com/jquery-migrate-1.2.1.min.js"></script>
<script type="text/javascript" src="slick/slick.min.js"></script>





    <script>
     
            $('.services_items__mobile').slick({
                infinite: true,
  slidesToShow: 3,
  slidesToScroll: 3
            });
      
    </script>

</body>
</html>